src/types/RelayPricer.ts

```diff
@@ -0,0 +1,10 @@
+export interface SourceApi {
+    query(input: string, ouput: string): Promise<number>;
```

Comment:
I would suggest using BigNumber for the return type as some ratios may be rather small (for example ratio of 1 iranian rial to btc is around 0.000000001100)

Reply:
I wanted to discuss before adding an extra bigNumber library but I did some test with bignumber.js and it works properly. I did not use BN since it does not work with decimals.

Reply:
updated